doc/release-notes/release-notes-1.8.0.md

```diff
@@ -0,0 +1,406 @@
+- Clean up and update the seeders list
+- Various improvements to deadlock detectors
+- Misc improvements to IBD
+- Use ctor to improve fetching tx from disc (txindex=1)
+- Add a priority queue for urgent message processing
+- Documentation update and improvements
+- Improve ElectrsCash integration
+- Rewrite of the fee estimator
+
+
+Features Details
+----------------
+
+### Increase length of unconfirmed transaction to 500 and mempool management improvements
+
+In previous release (1.7) we [improved](https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/release/doc/release-notes/release-notes-bucash1.7.0.md#new-cpfp-and-long-chains-of-unconfirmed-transactions) the code base to deal with chain of unconfirmed transactions in the mempool during the `getblocktemplate` work by 2 order of magnitudes. In this release O(n^2) complexity was [removed]( https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/2020) from the code that evict transactions from the mempool after they are included in a block that won the PoW lottery, and the [code]( https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/2032) that adds a transaction to a long chain in the mempool was optimized.
+
+That let us increase the maximum length on unconfirmed transactions accepted in the mempool to 500 without hitting any penalty performance wise.
+
+Increasing unilaterally such parameter is not something a single implementation, especially if not the most used, could do without some care. The problem is that this parameter is something we call "quasi-consensus" or in other words: there exists network wide configuration values that, if inconsistent, cause undesirable network behavior (but do not cause a fork).
+
+An example of this undesirable side effect is increasing the chance of success of double spend attacks. This is due to the fact that if BCH Unlimited accept, let's say, chain as long as 500 transactions whereas all the rest of the network set the limit to 50. To mitigate this side effect we implemented Intelligent unconfirmed transaction forwarding, i.e. when a block comes in that confirms enough parent transactions to make the transaction valid in non-BU mempools, a double-spender is essentially racing the entire BU network to push his double-spend into the miner nodes that now accept the transaction.
+
+Another unwanted side effect is the opening of new attack vector, which is usually called [reverse respend attack](https://gist.github.com/sickpig/231fbde2839a889d67848180d65f17b0#attack-scenario-2-reverse-respend). Basically the attacker leverage the discrepancy of network policy mixing unconfirmed UTXO with confirmed UTXO trying to be successful at double spending. This attack is completely solved by BCH Unlimited adding a constraint on transactions that has more than ABC max ancestor limits parents, which is that those could spend only one input, hence the attacker can mix UTXO, so no reverse respend attack can't be carried out.
+
```

Comment:
did you mean to say,  "can not mix UTXO"?

Reply:
yep, thanks will fix

Reply:
fixed